<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_3{
border-image: url(:/prefix1/snapshot.png);
}
#pushButton{
border-image: url(:/prefix1/Record.png);
}
#pushButton_2{
border-image: url(:/prefix1/closeRecord.png);
}
#pushButton_5{
border-image: url(:/prefix1/setting.png);
}
#pushButton_4{
border-image: url(:/prefix1/dianyuan.png);
}

#pushButton_3:pressed{
border-image: url(:/prefix1/snapshot1.png);
}
#pushButton:pressed{
border-image: url(:/prefix1/Record1.png);
}
#pushButton_2:pressed{
border-image: url(:/prefix1/closeRecord1.png);
}
#pushButton_5:pressed{
border-image: url(:/prefix1/setting1.png);
}
#pushButton_4:pressed{
border-image: url(:/prefix1/dianyuan1.png);
}
#PlayerBtn{
border-image: url(:/prefix1/player.png);
}
#PlayerBtn:pressed{
border-image: url(:/prefix1/player1.png);
}
#pushButton_6{
border-image: url(:/prefix1/small.png);
}
#pushButton_6:pressed{
border-image: url(:/prefix1/small1.png);
}
#centralwidget{
border-image:url(:/prefix1/border.png);
}</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>1000</width>
      <height>780</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1195</x>
      <y>460</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>400</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayerBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>400</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
